core/src/Streamly/Internal/Data/Fold.hs

```diff
@@ -683,12 +685,17 @@ countDistinctInt = fmap (\(Tuple' _ n) -> n) $ foldl' step initial
 -- identity (@0@) when the stream is empty. Note that this is not numerically
 -- stable for floating point numbers.
 --
--- > sum = fmap getSum $ Fold.foldMap Sum
+-- >>> sum = FoldW.cumulative FoldW.sum
+--
+-- Same as following but numerically stable:
+--
+-- >>> sum = Fold.foldl' (+) 0
+-- >>> sum = fmap Data.Monoid.getSum $ Fold.foldMap Data.Monoid.Sum
```

Comment:
You could just say "import Data.Monoid (getSum, Sum)" on the previous line, the fully qualified names can make the snippet less readable.

Reply:
They were fully qualified imports in other parts of the code. So I assumed it was like that for a reason.

Reply:
I guess this is fine. The imported module isn't too long.